packages/tables/resources/views/columns/image-column.blade.php

```diff
@@ -11,6 +11,7 @@ class="rounded-full bg-center bg-cover"
             ></div>
         @else
             <img
+                {!! $attributes->merge($getExtraImgAttributes()) !!}
```

Comment:
This will merge the main attribute bag onto the extra attributes

Check out the HasExtraAlpineAttributes trait - there's a method to get the `AttributeBag` which we can use here instead